src/SDKs/CognitiveServices/dataPlane/Search/Search/Microsoft.Azure.CognitiveServices.Search.csproj

```diff
@@ -1,12 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
   <PropertyGroup>
-    <PackageId>Microsoft.CognitiveServices.Search</PackageId>
+    <PackageId>Microsoft.Azure.CognitiveServices.Search</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services Search APIs.</Description>
     <VersionPrefix>1.0.0-preview</VersionPrefix>
     <DefineConstants>$(DefineConstants);CODESIGN</DefineConstants>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
These are redundant properties. We would like to control these properties from a central location.

Reply:
It's not clear what you're referring to, looks like the same comment as above? Are you saying we shouldn't include a PackageId? @DavidLiCIG are we not doing the right thing here

Reply:
synced offline, thanks for the clarification